src/code/z_camera.c

```diff
@@ -706,12 +788,12 @@ Vec3f Camera_CalcUpFromPitchYawRoll(s16 pitch, s16 yaw, s16 roll) {
 f32 Camera_ClampLERPScale(Camera* camera, f32 maxLERPScale) {
     f32 ret;
 
-    if (camera->atLERPStepScale < CAM_DATA_SCALED(R_CAM_AT_LERP_STEP_SCALE_MIN)) {
-        ret = CAM_DATA_SCALED(R_CAM_AT_LERP_STEP_SCALE_MIN);
+    if (camera->atLERPStepScale < R_CAM_AT_LERP_STEP_SCALE_MIN) {
```

Comment:
I don't like `CAM_DATA_SCALED` being inside `R_CAM_AT_LERP_STEP_SCALE_MIN` and similar, `R_` macros should read s16s (e.g. as set/shown in the reg editor), perhaps (?) expanding to constant defaults in NDEBUG

The options I see are remove the `R_` prefix or don't have `CAM_DATA_SCALED` in the macro
the former doesn't solve not knowing the type (and more importantly the unit/range) of the value
the latter sounds annoying for default values (may even not match)

Reply:
to bounce back on https://github.com/zeldaret/oot/pull/1917#discussion_r1513403713 ,
we could have the R_ regs in regs.h
and not-`R_`-prefixed macros with the default and `CAM_DATA_SCALED` in z_camera.c maybe
and eventually find a convention for indicating what is scaled or s16